components/brave_sync/client/bookmark_change_processor_unittest.cc

```diff
@@ -1466,3 +1467,110 @@ TEST_F(BraveBookmarkChangeProcessorTest, SyncTimestampMetaUpdateWay) {
   EXPECT_EQ(node_b_sync_timestamp,
       std::to_string(timestamp_resolve.ToJsTime()));
 }
+
+namespace {
+
+size_t g_overridden_minutes = 0;
+std::unique_ptr<base::subtle::ScopedTimeClockOverrides> OverrideForMinutes(
+    int overridden_minutes) {
+  g_overridden_minutes = overridden_minutes;
+  return std::make_unique<base::subtle::ScopedTimeClockOverrides>(
+    []() {
+     return base::subtle::TimeNowIgnoringOverride() +
+         base::TimeDelta::FromMinutes(g_overridden_minutes);
+    },
+    nullptr,
+    nullptr);
+}
+
+}  // namespace
+
+TEST_F(BraveBookmarkChangeProcessorTest, ExponentialResend) {
+  change_processor()->Start();
+  bookmarks::AddIfNotBookmarked(model(),
+                                 GURL("https://a.com/"),
+                                 base::ASCIIToUTF16("A.com"));
+
+  std::string send_retry_number;
+  GetSingleNodeByUrl(model(), "https://a.com/")->GetMetaInfo(
+     "send_retry_number", &send_retry_number);
+  EXPECT_TRUE(send_retry_number.empty());
+
+  int expected_send_retry_number = 0;
+
+  std::string record_a_object_id;
+
+  static const std::vector<int> exponential_waits = {0, 10, 20, 40, 80};
```

Comment:
can we have a static method for test to get this from bookmark_change_processor.cc instead of redefining it again?

Reply:
Yes, we can, but I made a duplication by intent to force look on failed testcase if the intervals would be changed.

Reply:
Fixed in https://github.com/brave/brave-core/pull/2986/commits/6ef48bbfe9d410087d0baf342e6bd641cbcd3c9b and https://github.com/brave/brave-core/pull/2986/commits/6c1bf1de0c0157bb2e0ffcd4cf915c8b05e3f15f